### ISSUE TITLE
Calling the “--status” flag with a LOGBASE

### ISSUE THREAD

**Issue by tschettervictor** [aaronhurt/zfs-replicate#44]
Since the FreeBSD port installs the application to $PATH, what’s the best way to call the “--status” command due to the fact that the default $PATH is not where the “config.sh” or the “logs” directory are stored?

You could do “LOGBASE=/path/to/logs zfs-replicate --status” or “zfs-replicate -s /path/to/config.sh”
Both work.

**Comment by aaronhurt** [comment]
They do both work, and all of these _should_ work, but I think some make more sense than the others.

These make sense using environment variables.
```
LOG_BASE=/path/to/logs zfs-replicate.sh --status
```
```
LOG_BASE=/path/to/logs zfs-replicate.sh -s
```

Putting the config first in a historic fashion makes sense, but it's odd syntax.
```
zfs-replicate.sh /path/to/config.sh --status
```
```
zfs-replicate.sh /path/to/config.sh -s
```

This really doesn't make sense at all. It works, but is really awful syntax. The config looks like an argument to the status flag.
```
zfs-replicate.sh --status /path/to/config.sh
```
```
zfs-replicate.sh -s /path/to/config.sh
```

If you want to pass the config to the command you _should_ use an option for both the config and status.
```
zfs-replicate.sh --status --config /path/to/config.sh
```
```
zfs-replicate.sh -s /path/to/config.sh -c /path/to/config.sh
```

You can also mix and match `-s` with `--config` and `--status` with `-c`, but the point being leaving the config dangling is odd syntax and is only supported for historical reasons.

**Comment by aaronhurt** [comment]
We shouldn't allow dangling configs, I've fixed this in #45
